### DISCUSSION TITLE
Add an option to kill on combat logging

### DISCUSSION THREAD

**Discussion by Recliness** [Zrips/CMI#8131] [Enhancement]
This would discourage players from logging out during combat

**Comment by CloudeLecaw** [comment]
isnt this a bit out of scrope of CMI? it would need CMI to create a dummy player when combat is tunred of that got killed and drops all of players inventory.  + clear players inventary and lvls when he relog.

**What you can do currently:**
- set a usermeta when the combar starts - and remove the meta when combat ends.
  (EventCommands.yml - pvpstart &  pvpend)
- When player logos out in combat nothing would happen here.
- when player relogs he gets killed and meta would be removed. (EventComamnds.yml joinServer, playerDeath, playerKillPlayer)
Having meta here means - player is in battle so you can set a meta like a marker that player is in battle and markers (metas) can be checked in EventCommands.yml events.)

Note: currently it would not make sense to kill the loged out player with quitServer event because he allready is off when he left and current kill command and only kill a player that is online - what is not the case when the player logged of.
Im not sure if CMI wants to add a dummy npc for that case that stays for a few secounds with equipment and stats of player.
Thats what other plugins like PvPManager do - they create an NPC dummy copy of the player. The Dummy gots killed and the player inventory and stats got cleared.

**Reply by Recliness** [reply]
What exactly is the scope of cmi? Also why have a combat tagging system without combat logging? It could just add them to a list of people to kill on join rather than creating a dummy npc like you were saying

**Reply by CloudeLecaw** [reply]
cmi never was in scrop of pvp managing the current combat including feature is just a little one with minimal function that was added to have something instead of nothing. but if you wanne have it more extanted you should switch to one of the pvp plugins that have this as a scope.

**Reply by CloudeLecaw** [reply]
at lease the disuccsion here are for everyone to request everything they want  :-D
it was just more like an info, that you dont need to aspect it also will be added in a near future. pvp management has a very very low priority in scope of cmi
